packages/rum-core/src/domain/trackEventCounts.spec.ts

```diff
@@ -27,11 +27,12 @@ describe('trackEventCounts', () => {
     notifyCollectedRawRumEvent({ type: RumEventType.LONG_TASK })
     expect(eventCounts.longTaskCount).toBe(1)
   })
-
-  it("doesn't track views", () => {
-    const { eventCounts } = trackEventCounts({ lifeCycle, isChildEvent: () => true })
-    notifyCollectedRawRumEvent({ type: RumEventType.VIEW })
-    expect(objectValues(eventCounts as unknown as { [key: string]: number }).every((value) => value === 0)).toBe(true)
+  ;[RumEventType.VIEW, RumEventType.VITAL].forEach((eventType) => {
```

Comment:
❔ **Question**: Why using the `;` at the start of this line 🤔 ?

Reply:
it is actually a syntax requirement.
FMU, when not using trailing semi-colon style, you have to put a semi colon before square brackets to differentiate:
```js
obj[x]

// VS

obj;[x]
```